cli/src/search.rs

```diff
@@ -140,92 +166,238 @@ pub fn search_files<P: AsRef<Path>>(path: P) -> anyhow::Result<Vec<Test>> {
 #[cfg(test)]
 mod tests {
     use super::search_files;
+    use kythera_lib::{to_vec, Abi, Method};
     use std::fs::{self, File};
+    use std::io::Write;
+    use std::path::Path;
     use tempfile::tempdir;
 
+    // Util to create actor files on a given directory path.
+    fn set_actors_in_dir(dir_path: &Path, actors: Vec<(&str, &Abi)>) -> () {
```

Comment:
NIT: sorry didn't notice this one.
```suggestion
    fn create_actors_in_dir(dir_path: &Path, actors: Vec<(&str, &Abi)>) -> () {
```